tests/tests_unit/rules/test_models/test_information_rules.py

```diff
@@ -216,51 +327,15 @@ def test_david_as_dms(self, david_spreadsheet: dict[str, dict[str, Any]]) -> Non
 
         assert isinstance(dms_rules, DMSRules)
 
-    def test_olav_as_dms(self, olav_rules: InformationRules) -> None:
```

Comment:
Why is this test removed? It is checking that references are correctly set when converting Olav's information spreadsheet.

Reply:
As far as I can see there is nothing that replaces this test to do the same check.